### DISCUSSION TITLE
mcbbs.net真的要没了吗

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3379] [提问] [answered]
我看到一些内部消息说mcbbs将永久关停，并且不久后就会发公告，这是真的吗？（dc那有人说龙猫说是真的）

**Comment by Deep-Dark-Forest** [comment]
在发布公告之前，没人能确定是否是真的。不过一旦是真的，我们可能确实需要一个新下载源了……

**Reply by hejiehao** [reply]
HMCL 已经删掉 MCBBS 源了

**Reply by Deep-Dark-Forest** [reply]
> HMCL 已经删掉 MCBBS 源了

~~你果然是那个项目的成员哈哈~~

**Comment by allMagicNB** [comment, marked as answer]
龙猫发的动态也没说是可能啊，是肯定的语气

**Comment by Deep-Dark-Forest** [comment]
【【最新！】MCBBS永久关站？最全属实资料-哔哩哔哩】 https://b23.tv/iWeF00q

**Comment by WForst-Breeze** [comment]
肯定是关了的

**Comment by wuliaodexiaoluo** [comment]
要关了，并且现在有没有备份还是个问题
